### PR TITLE
Fellow search

### PR DESCRIPTION
**Pull request by bellmyer**
Fellow search is implemented using pg_search. Fellows can be searched by first name, last name, or e-mail fragment. This originally used the sunspot gem tied to a solr search instance, but the user interface remains identical.

Fellows are searchable on the admin/fellows page, via a simple form.

**screencap:** https://vimeo.com/310578570/306a767db8

20190110-fellow-search

### CODE REVIEW

**Comment by adamdruppe** [comment]
You've gotta be kidding me. 1000 lines of config stuff, several gems, and a major external dependency to do a *basic name search*?

**Comment by bellmyer** [comment]
@adamdruppe thanks for the tip about pg_search, which I've implemented in place of sunspot/solr. For what it's worth, there are more respectful ways to tell a colleague you might have a better solution to their problem.

File: Gemfile.lock [hunk 1]
```diff
@@ -321,6 +332,12 @@ GEM
     sshkit (1.17.0)
       net-scp (>= 1.1.2)
       net-ssh (>= 2.8.0)
+    sunspot (2.3.0)
```
Comment (sadleb) [inline Gemfile.lock]: Should we remove these gems now that they are not used?

Comment (bellmyer) [reply Gemfile.lock]: Hi Brian - I ran `bundle install` to clean up Gemfile.lock. It was a docker oversight on my part that the old version of the lock file was sticking around. Shouldn't be a problem in the future. Thanks for the catch!

File: Gemfile.lock [hunk 2]
```diff
@@ -390,6 +408,7 @@ DEPENDENCIES
   split
   spring
   spring-watcher-listen (~> 2.0.0)
+  sunspot_rails
```
[the remark this answers is not in the document]
Comment (bellmyer) [reply Gemfile.lock]: this is removed as well :)

**Pull request merged**